src/main/java/com/google/api/codegen/util/CommonRenderingUtil.java

```diff
@@ -112,4 +112,9 @@ public static String padding(int width) {
   public static int toInt(String value) {
     return Integer.valueOf(value);
   }
+
+  /** Returns the lowercase version of the given text */
+  public static String lowercase(String value) {
```

Comment:
Is this method needed?

Reply:
Is there a better way to lowercase a string in a snip file (like a built-in)?

I didn't want to add another property on the model because the capitalized version is used for other things and two properties for the version seems odd.

Reply:
`{@apiMethod.apiVersion.toLowerCase}`

Reply:
Done. Thanks for the pointer!